packages/webpack-cli/lib/groups/resolveConfig.js

```diff
@@ -179,7 +179,7 @@ const finalize = async (moduleObj, args) => {
 
     if (configName) {
         const foundConfigNames = [];
-        const configsToFilter = isFunctionalConfig ? configs[0] : configs;
+        const configsToFilter = Array.isArray[configs[0]] ? configs[0] : configs;
```

Comment:
I still think we have problems here, maybe `Array.isArray` need here

Reply:
If nothing else, the square brackets are a clear typo: `Array.isArray[…]` → `Array.isArray(…)`.